.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
+And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
+## Issues fixed by this PR
+
+<!--- Put the links of issues that fixed by this PR here (if any). -->
+## Type of changes
+
+<!--- Put an `x` in all the boxes that apply your changes. -->
+
+- [ ] Bug fix
+- [ ] New feature 
+- [ ] Enhancement
+- [ ] Documentation
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
+- [ ] I am responsible for any copyright issues with my code if it occurs in the future.
```

Comment:
- [ ] My pull request is unique and no other pull requests have been opened for these changes
- [ X ] I agree that crepes are better than pancakes

Reply:
![lmao](https://media.discordapp.net/attachments/954096375522549782/968140348230627378/Screen_Shot_2022-04-25_at_8.23.16_PM.png)